web/src/main/java/org/akaza/openclinica/web/crfdata/ImportCRFDataService.java

```diff
@@ -497,7 +497,21 @@ public SummaryStatsBean generateSummaryStatsBean(ODMContainer odmContainer, List
                                         blankCheck.put(newKey, itemDataBean);
                                         logger.info("adding " + newKey + " to blank checks");
                                         if (!metadataBeans.isEmpty()) {
-                                            ItemFormMetadataBean metadataBean = metadataBeans.get(0);
+                                        	
+                                            ItemFormMetadataBean metadataBean = null;
+                                           //OC-14979
+                                        	for(int i=0; i<metadataBeans.size();i++ ) {
+                                        		ItemFormMetadataBean metadataBeanTemp = metadataBeans.get(i);
+                                        		
+                                        		if(metadataBeanTemp.getCrfVersionId() == crfVersion.getId()) {
+                                        			metadataBean = metadataBeanTemp;
+                                        			break;
+                                        		}
+                                        	}
+                                        	
+                                        	if(metadataBean == null) {

```

Comment:
In this case, we should throw an error instead of returning the first element

Reply:
created a new PR #3462